app/javascript/app/components/sectors-agriculture/countries-context/meat-data/tooltip/tooltip-component.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Tag } from 'cw-components';
+import { get } from 'lodash';
+
+import { format } from 'd3-format';
+
+import styles from './tooltip-styles.scss';
+
+const Tooltip = ({ content, config }) => {
+  const payload = get(content, 'payload[0].payload');
+  const category = (payload && payload.x) || '';
+
+  const countryValue = payload && payload.yCountry;
+  const otherCountriesValue = payload && payload.yOthers;
+
+  const countryName = get(config, 'tooltip.yCountry.label');
+  const othersName = get(config, 'tooltip.yOthers.label');
+
+  const countryColor = get(config, 'theme.yCountry.stroke') || '';
+  const othersColor = get(config, 'theme.yOthers.stroke') || '';
+  const unit = get(config, 'axes.yLeft.unit') || '';
+
+  return (
+    <div className={styles.tooltip}>
+      <div className={styles.category}>{category}</div>
+      <p className={styles.info}>
+        <Tag
```

Comment:
I think you can extract <Tag /> component to separate method `renderTag()` and pass name, value and color properties.